fog/view/server/src/fog_view_router_service.rs

```diff
@@ -69,11 +65,11 @@ where
             let logger = logger.clone();
             // TODO: Confirm that we don't need to perform the authenticator logic. I think
             // we don't  because of streaming...
```

Comment:
Total aside: Could you clarify this TODO?

Reply:
I'm pretty sure that the authenticator enforces unary authentication logic, so I don't think it applies for streaming. I can look more into this though, but will make changes in a follow-up PR if necessary.

Reply:
Yeah, I guess what I'm saying is: what logic would go in here?